src/pages/ProductPage.jsx

```diff
@@ -17,6 +17,8 @@ import {
 } from "@fortawesome/free-solid-svg-icons";
 import AboveProductTitle from "../components/AboveProductTitle/AboveProductTitle";
 import Reviews from "../components/Reviews/Reviews";
+import MeetYourHost from "../components/MeetYourHostRight/MeetYourHostRight";
```

Comment:
You just imported MeetYourHost, but did not use it in this component. So, I think it is better to remove it from here.